mod_ci/controllers.py

```diff
@@ -45,6 +52,9 @@ class Status:
 
 @mod_ci.before_app_request
 def before_app_request():
+    """
+    Function tht organize menu content such as Platform mgmt 
```

Comment:
Please be aware of typo's. You also don't need to abbreviate management in the documentation ;)